Fakes/Fakes/Fakes.generated.swift

```diff
@@ -1360,3 +1381,17 @@ extension UploadableMedia {
         )
     }
 }
+extension WCPayAccountStatusEnum {
```

Comment:
Why these changes?

Reply:
I think that someone added the `GeneratedFakeable` conformance to those models, but never generated the Fakes extensions. :nod: